src/pages/BirthDay.jsx

```diff
@@ -1,11 +1,48 @@
 import Layout from "../components/Layout/Layout";
+import CakeIcon from "@mui/icons-material/Cake";
+import { styled, Stack, Typography } from "@mui/material";
 
 const BirthDay = () => {
   return (
     <Layout>
-      <div>BirthDay</div>
+      <BirthdayWrapper>
+        <TitleWrapper>
+          <CakeIcon sx={{ fontSize: 40 }} />
+          <Typography fontSize="20px">생일!</Typography>
+        </TitleWrapper>
+        <MonthChip>
+          <Typography>MonthChip</Typography>
+        </MonthChip>
+        <BirthdayList>
+          <Typography>생일자 명단</Typography>
+        </BirthdayList>
+      </BirthdayWrapper>
     </Layout>
   );
 };
 
+const BirthdayWrapper = styled(Stack)`
+  height: calc(100dvh - 120px);
+  width: 100vw;
+`;
+const TitleWrapper = styled(Stack)`
+  flex-direction: row;
+  align-items: flex-end;
+  width: 35%;
+  height: 10  %;
+  justify-content: space-evenly;
+`;
```

Comment:
```suggestion
const TitleWrapper = styled(Stack)`
  flex-direction: row;
  align-items: center;
  height: 10%;
  gap: 20px;
  padding: 0 10px;
`;
```
왠만하면 width로 지정하기 보단 margin이나 padding을 잘 이용하도록 하고 `gap`을 이용하면 flex 요소 자식 태그간의 간격 너비를 지정할 수 있으니까 활용하면 좋을듯

Reply:
@DHyeonJ @dony0720 둘다 참고
그리고 참고로 Stack 요소의 경우는 아래와 같은 부분이 가능하고, 매우 유용하니까 참고

```
<TitleWrapper gap={2} mx={5}> example </TitleWrapper>
```
gap은 아래 자식요소간 간격
p={1} 로하면 패딩 상하좌우 전체 1만큼
px={1} 로 하면 좌우에 1만큼
py={1} 은 상하에 패딩 1만큼
같은 맥락으로 pb, pt, pl, pr (아래,위,왼쪽,오른쪽) 가능 (padding-bottom, padding-left 같은것들의 축약이라고 보면됨)

마진도 동일하게 m, my, mx, mb 등으로 사용 가능. 

아직 마진 패딩의 개념을 잘 모른다면 문제있는거니까 모르면 찾아서 확실히 이해해

Reply:
네넵 알겠습니다!!!